README.md

```diff
@@ -103,21 +105,46 @@ $ sudo dnf -y install nodejs
 > Don't forget to install node-cli, node-gyp packages using `$ npm install -g node-cli node-gyp` if it's not installed.<br />
 > Don't forget to do `npm init` to generate profile.json.
 
+## Platform Setup
+
 ### iOS
 
-Run Mobidex from XCode to use the simulator. First do the following:
+> Note: Can only be done on a Mac OS X Specific Setup
+
+Additional Requirements
+
+- [bundler](https://bundler.io)
+
+To run Mobidex on the iOS simulator from XCode, follow these steps.
+
+1. From the Mobidex root  directory, install all the required javascript packages.
+
+  `npm install`
+
+2. Install all the required gems in your project space.  
+
+  > This command will change to the iOS working directory to setup the required Ruby gems in your local project.
+
+  `pushd ios && bundle install --path .gems && bundle exec pod install && popd`
+
+3.  Open `ios/mobidex.xcworkspace` in XCode
+
+4.  Select **Build Settings** for `Pods > secp256k1_swift` target and do the following:
+    > Tip: You'll notice there are 2  pods with similar names, we are only working with the one that has the underscore 'secp256k1_swift'
+
+    - Find and set **Use Header Maps** to "No"
+    - Find **Header Search Paths**
```

Comment:
One of the below should be `User Header Search Paths` I believe.

Reply:
For two options exist.  

- Header Search Paths --> is used for frameworks/libraries regardless of system framework or 3rd party
- User Header Search Paths --> is for project headers being compiled

Only, "Header Search Paths" only worked for me.  When using "User Header Search Paths" I get an error... 
`secp256k1.h' file not found with <angled>...`